asv/commands/publish.py

```diff
@@ -171,6 +171,14 @@ def copy_ignore(src, names):
             for results in cls.iter_results(conf, repo, range_spec):
                 log.dot()
 
+                branches_for_commit = [branch for branch, commits in branches.items() if
+                            results.commit_hash in commits]
+
+                # Print a warning message if we couldn't find the branch of a commit
+                if not len(branches_for_commit):
```

Comment:
`len()` is useless here.

Reply:
I figured this was more explicit that way, it's almost a question of code style. If you think it's cleaner, I'll remove it

Reply:
len() is not recommended